81.md

```diff
@@ -0,0 +1,38 @@
+NIP-81
+======
+
+Private Relationship Status
+---------------------------
+
+`draft` `optional`
+
+A new `kind:30382` event documents a "Private Relationship Status" between two users of the network. 
+
+It is defined as a _parameterized replaceable event_ with a single `d` tag as a random UUID per pubkey. An optional `n` tag adds the target key to an unbound list.
+
+The other tags are stringified, NIP-44-encrypted and placed inside the `.content` of the event. 
+
+For example:
+
+```js
+{
+  "kind": 30382,
+  "tags": [
+    ["d", "<randomUUID>"],
+    ["n", "Clients"]
```

Comment:
What do you think of defining a NIP-51 list kind to hold all possible statuses? It would be easier to help user pick
a status label when adding/editing a contact. It would be possible to set the `n` tag to a private status ("6064460175057025" instead of "Idiot").

```js
{
  "kind": 1XXXX,
  "content": "<nip44Encrypt(JSON.stringify([
    ["status", "<status 1>", "random string 1"],
    ["status", "Clients", "Clients"] // public
    ["status", "Idiot", "6064460175057025"] // private
  ]))>",
  // ...other fields
}
```

Reply:
I think `n` itself can be public or private. Clients can always show the `n` value if that value is not found in the kind 10XXXX list

Reply:
ohh interesting.. I will add it to the 10xxx events as well